### PR TITLE
Update polish localization (once again)

### PR DESCRIPTION
**Pull request by MeTonaTOR**
As of title, new translations were added, therefore they are now translated aswell

### CODE REVIEW

File: RetroBar/Languages/polski.xaml [hunk 1]
```diff
@@ -40,6 +41,11 @@
     </x:Array>
     <s:String x:Key="version">Wersja {0}</s:String>
     <s:String x:Key="visit_on_github">Zobacz na GitHub</s:String>
+    <s:String x:Key="taskbar_scale">Skala Taskbar</s:String>
+    <s:String x:Key="taskbar_scale_1x">100%</s:String>
+    <s:String x:Key="taskbar_scale_2x">200%</s:String>
+    <s:String x:Key="taskbar_scale_current">Aktualne ustawienie: {0}%</s:String>
+    <s:String x:Key="debug_logging">Enable debug logging</s:String>
```
Comment (xoascf) [inline RetroBar/Languages/polski.xaml:48]: Missing translation?

File: RetroBar/Languages/polski.xaml [hunk 2]
```diff
@@ -53,30 +59,48 @@
...
     <s:String x:Key="show_hidden">Pokaż ukryte ikony</s:String>
     <s:String x:Key="hide">Ukryj</s:String>
 
     <s:String x:Key="open_folder">&amp;Otwórz folder</s:String>
-</ResourceDictionary>
\ No newline at end of file
+
+    <!-- Items below are settings, and should always be in English ( and they will :P ) -->
+    <FlowDirection x:Key="flow_direction">LeftToRight</FlowDirection>
+    <FontFamily x:Key="start_text_xp_font_family">Franklin Gothic</FontFamily>
```
Comment (xoascf) [inline RetroBar/Languages/polski.xaml:105]: There is no need to add this if you are not going to change anything here.

**Comment by MeTonaTOR** [comment]
Alright, gonna fix them asap

pon., 29 maj 2023, 16:03 użytkownik Amaro Martínez ***@***.***>
napisał:

File: RetroBar/Languages/polski.xaml [hunk 3]
```diff
@@ -53,30 +59,48 @@
     <s:String x:Key="start_text">Start</s:String>
     <s:String x:Key="start_text_xp">Start</s:String>
     <s:String x:Key="start_button_tip_98">Kliknij tutaj aby rozpocząć.</s:String>
-    <s:String x:Key="start_button_tip">Kliknij, aby rozpocząć</s:String>
+    <s:String x:Key="start_button_tip">Kliknij tutaj aby rozpocząć</s:String>
```
Comment (xoascf) [inline RetroBar/Languages/polski.xaml:62]: XPPol
The previous translation seems to be more accurate, why did you change it?

**Comment by MeTonaTOR** [comment]
Access keys and missing translation has been fixed

**Comment by xoascf** [comment]
This is perfect, thanks!
